### PR TITLE
[Do Not Merge]Putsubscription api

### PR DESCRIPTION
**Pull request by navysingla**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by Juliehzl** [comment]
@navysingla You don't want to merge this PR, right? I add [DoNotMerge] label for your PR. And before reviewing, can you try to fix errors in CI checks first?

File: specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-03-01-preview/subscriptions.json [hunk 1]
```diff
@@ -147,6 +147,73 @@
...
+        },
+        "x-ms-long-running-operation": true,
+        "responses": {
+          "200": {
+            "description": "Normal response for a successful query. The response body will contain the data that matches the filters specified in the query parameters.",
+            "schema": {
+              "$ref": "#/definitions/SubscriptionCreationResult"
+            }
+          },
+          "202": {
```
Comment (KrisBash) [inline specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-03-01-preview/subscriptions.json]: The 202 pattern is fine, but generally 201 is preferred for resource create. https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#asynchronous-operations

**Review by KrisBash — CHANGES_REQUESTED** [review]
Please see comments. Feel free to reach out with any questions

**Comment by yungezz** [comment]
hi @navysingla are you still on this PR? Could you pls address comments? Is it ok to close if you're not ready for reviewing?

**Comment by navysingla** [comment]
I will work on them tomorrow

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by yungezz** [comment]
hi @navysingla ping for update? Is it ok to close it for now since no update in 29 days?

**Comment by yungezz** [comment]
hi @Juliehzl, could you pls follow up on this PR?

**Comment by akning-ms** [comment]
Hi @navysingla, can we close this PR as it is idle for a long time.

**Comment by ravbhatnagar** [comment]
Removing ARM labels for a 1 year old PR.

**Comment by Juliehzl** [comment]
Hi  @navysingla, do you still need the PR? If yes, please fix CI error and remove donotmerge.

**Comment by Juliehzl** [comment]
close the PR because there is no response for a long time. If you are ready for review, feel free to reopen it.

**Pull request closed without merging**